aya/src/obj/btf/types.rs

```diff
@@ -150,10 +154,9 @@ impl BtfType {
 
         use BtfType::*;
         match self {
-            Unknown => 0,
-            Fwd(_) | Const(_) | Volatile(_) | Restrict(_) | Ptr(_) | Typedef(_) | Func(_) => {
-                ty_size
-            }
+            Unknown => ty_size,
```

Comment:
As discussed on discord, to fix the infinite loop on `Unknown` bug, this should be `Unknown => ty_size`

Reply:
fixed in https://github.com/alessandrod/aya/pull/35/commits/4465131199e36f288c552a9cbc609fa6720a3d2f